view/rooms/new/app.js

```diff
@@ -54,6 +55,28 @@ function getPathVariables(adr){
   };
 }
 
+function createRoom(paramaters){
+  const req = request.post('/v1/rooms');
+  const obj = {
+    name: paramaters.name,
+    type: paramaters.type,
+    externalId: paramaters.externalId,
+    active: paramaters.active,
+  };
+  req
+    .send(obj)
+    .end((error, res) => {
+      if (error) {
+        if (error.response.text.includes('SequelizeUniqueConstraintError')) {
+          window.location.href = `/rooms/${paramaters.name}`;
+        }
+        console.error(error.response.text);
+      } else {
+        window.location.replace(`/rooms/${res.body.id}`);
```

Comment:
can we keep URL params? maybe keeping params should be explicitily requested by adding `&keepParams=true`?